### PR TITLE
Updated uuid generation and clone repo link (#5)

### PR DESCRIPTION
**Pull request by zashraf1985**
### Description

1. Deploy url was pointing to the fork. Updated that to point to the main repository.
2. user id generation was being done using a naive 6 digit random number generator. Changed it to use `crypto.randomUUID()` library instead.

### Best Way to Test

1. Try the `Deploy` button and verify the repository is being cloned from the main repo and not the fork.
2. Run the application and check edgeworker logs. The user id should be uuid instead of a 6 digit integer id.

### Type of Change

- [x] Bug fix (non-breaking change that fixes an issue)
- [  ] New Example
- [x] New feature in existing example

### New Example Checklist

- [x ] 🚀 Link to deployment URL on Vercel
- [x] 📱 Is it responsive? Are mobile and tablets considered?
- [x] 📚 `README.md` preview link in PR description (branch)
- [x] ⚙️ Secrets have instructions for how to set up in the readme
- [  ] 🛫 `npm run new-example` was run to create the example

### CODE REVIEW

File: edge-functions/feature-flag-optimizely/pages/_middleware.tsx [hunk 1]
```diff
@@ -1,22 +1,19 @@
...
-}
-
 export async function middleware(req: NextRequest, ev: NextFetchEvent) {
   if (!req.page.name) {
     return NextResponse.next()
   }
 
   // Fetch user Id from the cookie if available so a returning user from same browser session always sees the same variation.
-  const userId = req.cookies[COOKIE_NAME] || generateRandomUserId()
+  const userId = req.cookies[COOKIE_NAME] || v4()
```
Comment (lfades) [inline edge-functions/feature-flag-optimizely/pages/_middleware.tsx:16]: Can we switch this to use [crypto.randomUUID()](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID)?

**Review by lfades — CHANGES_REQUESTED** [review]
@zashraf1985 Hi there and thank you for the PR!

I added a comment about preferring the use of the crypto web API over the uuid package.

File: edge-functions/feature-flag-optimizely/pages/_middleware.tsx (hunk 1 above)
Comment (zashraf1985) [reply edge-functions/feature-flag-optimizely/pages/_middleware.tsx]: Done!

**Comment by zashraf1985** [comment]
@lfades I have incorporated the changes you asked for. It will be awesome if you can check and merge this one.

**Review by lfades — APPROVED** [review]
Thank you!

**Pull request merged**